pkg/pushsync/pushsync.go

```diff
@@ -189,13 +187,6 @@ func (ps *PushSync) pushToClosest(ctx context.Context, ch swarm.Chunk) (rr *pb.R
 
 		defersFn()
 
-		deferFuncs = append(deferFuncs, func() {
```

Comment:
this is needed since we might be logging the error unnecessarily when the error is returned from `ClosestPeer` below